gazebo_plugins/src/gazebo_ros_camera_utils.cpp

```diff
@@ -231,13 +232,13 @@ void GazeboRosCameraUtils::Load(sensors::SensorPtr _parent,
   else
     this->distortion_t2_ = this->sdf->Get<double>("distortionT2");
 
-  if ((this->distortion_k1_ != 0.0) || (this->distortion_k2_ != 0.0) ||
-      (this->distortion_k3_ != 0.0) || (this->distortion_t1_ != 0.0) ||
-      (this->distortion_t2_ != 0.0))
+  if (!this->sdf->HasElement("borderCrop"))
```

Comment:
it would be nice to document the new `borderCrop` parameter somewhere, perhaps in your new [ROS depth camera](http://gazebosim.org/tutorials?tut=ros_depth_camera&branch=ros_depth_camera_preview) tutorial or the [Gazebo plugins in ROS tutorial ](http://gazebosim.org/tutorials?tut=ros_gzplugins&cat=connect_ros)